### DISCUSSION TITLE
Welcome to actframework Discussions!

### DISCUSSION THREAD

**Discussion by greenlaw110** [actframework/actframework#1371] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by DingouerFish** [comment]
Hi,I'm ShenTao I'm using ActFramework to develop my new project, so far so good. ActFramework is great, it deserves more attensions. I have introduced it to some friends. Thanks for your great work.  @greenlaw110

**Reply by greenlaw110** [reply]
Thanks @DingouerFish. I hope you enjoy working with Act and feel free to drop questions here anytime.
